Bottom: 2f56e0464b591a6fd34365960ff90c801d283d20
Top:    d89b3b17143e4f1cbb44aba06931cc25f912142b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 01:56:06 +0000

gtkhid: Return NULL from ghid_get_net_from_node_name()

Fixes calling return; from function returning non-void type.




---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 0048fcf..3696c8f 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -865,7 +865,7 @@ ghid_get_net_from_node_name (gchar * node_name, gboolean enabled_only)
 
   /* If no netlist is loaded the window doesn't appear. */
   if (netlist_window == NULL)
-    return;
+    return NULL;
 
   task.enabled_only = enabled_only;
   task.node_name = node_name;
